Lib/test/test_tokenize.py

```diff
@@ -1849,8 +1849,8 @@ def test_random_files(self):
         # pass the '-ucpu' option to process the full directory.
 
         import glob, random
-        fn = support.findfile("tokenize_tests.txt")
-        tempdir = os.path.dirname(fn) or os.curdir
+        fn = support.findfile("tokenize_tests.txt", subdir='tokenizedata')
+        tempdir = os.path.dirname(os.path.dirname(fn)) or os.curdir
```

Comment:
I'm not sure about the logical to get tempdir here. Maybe start from __file__ instead? Why is it called tempdir? That's Lib/test/, the directory of test files, no?

Reply:
This is not something I am modifing here. I only changed 
`tempdir = os.path.dirname(fn) or os.curdir` 
to be
`tempdir = os.path.dirname(os.path.dirname(fn)) or os.curdir`

I can open a new issue about it and dig into it later :)